lib/propolis-datas/src/apikey.rs

```diff
@@ -60,7 +60,7 @@ pub struct ApiKey {
 #[async_trait]
 pub trait ApiKeyStore {
     /// Store the item inside the particular DB
-    async fn store(&mut self, item: &ApiKey) -> anyhow::Result<ApiKey>;
+    async fn store(&self, item: &ApiKey) -> anyhow::Result<ApiKey>;
```

Comment:
Hmm, das `&mut` ist da eigentlich für den Fall drin, dass ich nicht den SqlitePool, sondern einen anderen Store nutzen will, welcher internen zustand hält (wie z.B. eine HashMap). Bin mir gerade nicht sicher, wie das ohne dass &mut gehen soll

Reply:
Clippy hat gesagt, dass das gerade nicht nötig ist. Entweder wir treffen das und deaktivieren die Warning für diesen code, oder wir führen es wieder ein, wenn es gebraucht wird.

Reply:
Eigentlich dürfte das gar nicht kompilieren, denn die Tests in Zeile 120 nutzen das Feature. Kann es sein, dass wir gar nicht die Tests der Workspaces ausführen?

Reply:
Gute Frage, könnte sein